src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let count = 0;
+
+  const iterate = (figure) => {
+    if (figure !== undefined) {
+      count += figure;
+
+      return iterate;
+    } else {
```

Comment:
Redundant else statement here. You have a return above 
```suggestion
    }
```